scripts/rff/historical_weather_variability/4km/places_4km.tmcf

```diff
@@ -0,0 +1,6 @@
+Node: dcid:WeatherVariability_Counties->dcid
```

Comment:
Looking at the summary_report.html:

https://htmlpreview.github.io/?https://github.com/datacommonsorg/data/blob/511710fd9f6b7066090c66216f3f3b4e28355077/scripts/rff/historical_weather_variability/4km/dc_generated/summary_report.html

It seems the dates end with 1997. Is that just a sample?

Reply:
yes, that's just the sample